scripts/generate-releases-manifest.mjs

```diff
@@ -135,21 +135,34 @@ if ( isDevBuild ) {
 } else {
 	console.log( 'Adding latest release ...' );
 
-	const releaseVersionZipFilename_x64 = `https://cdn.a8c-ci.services/studio/studio-${ process.platform }-x64-v${ version }.app.zip`;
-	const releaseVersionZipFilename_arm64 = `https://cdn.a8c-ci.services/studio/studio-${ process.platform }-arm64-v${ version }.app.zip`;
-	// const releaseVersionZipFilename_win32 = `https://cdn.a8c-ci.services/studio/studio-${ process.platform }-arm64-v${ version }.app.zip`;
+	const releaseVersionZipFilename_mac = `https://cdn.a8c-ci.services/studio/studio-darwin-v${ version }.app.zip`;
+	const releaseVersionZipFilename_x64 = `https://cdn.a8c-ci.services/studio/studio-darwin-x64-v${ version }.app.zip`;
+	const releaseVersionZipFilename_arm64 = `https://cdn.a8c-ci.services/studio/studio-darwin-arm64-v${ version }.app.zip`;
+	const releaseVersionZipFilename_win32 = `https://cdn.a8c-ci.services/studio/studio-win32-v${ version }.exe`;
```

Comment:
Interesting mix of camel- and snake- casing.

Did you use it to better visually distinguish the platform?

Did you consider:

```suggestion
	const releaseVersionZipFilenameMac = `https://cdn.a8c-ci.services/studio/studio-darwin-v${ version }.app.zip`;
	const releaseVersionZipFilenameX64 = `https://cdn.a8c-ci.services/studio/studio-darwin-x64-v${ version }.app.zip`;
	const releaseVersionZipFilenameArm64 = `https://cdn.a8c-ci.services/studio/studio-darwin-arm64-v${ version }.app.zip`;
	const releaseVersionZipFilenameWin32 = `https://cdn.a8c-ci.services/studio/studio-win32-v${ version }.exe`;
```

Reply:
Addressed in [ed8c467](https://github.com/Automattic/studio/pull/114/commits/ed8c4679f15e554955a7c768421f6597a6da3faf)